tests/bin/external_trogdor_command_example.py

```diff
@@ -20,19 +20,22 @@
 
 #
 # This is an example of an external script which can be run through Trogdor's
-# ExternalCommandWorker.
+# ExternalCommandWorker. It parses the custom workload JSON and sleeps for the given amount of time.
```

Comment:
Maybe "It sleeps for the given amount of time expressed by the delayMS field in the ExternalCommandSpec.